templates/compose/gotify.yaml

```
@@ -0,0 +1,31 @@
+# documentation: https://gotify.net/docs/install
+# slogan: Gotify is an open-source self-hosted notification server.
+# category: productivity
+# tags: productivity,notification,collaboration
+# logo: svgs/gotify.png
+# port: 80
+
+services:
+  gotify:
+    image: gotify/server
+    ports:
+      - 8080:80
+    environment:
+      - SERVICE_FQDN_GOTIFY_80
```

Use `SERVICE_URL_GOTIFY_80` instead of `SERVICE_FQDN_GOTIFY_80` because `SERVICE_FQDN_GOTIFY_80` won't include `http://` protocol
```suggestion
      - SERVICE_URL_GOTIFY_80
```